test/citestutils.js

```diff
@@ -62,6 +63,24 @@ function testBackup(params, databaseName, outputStream, callback) {
     }
   }
 
+  // Pipe via encryption if requested
+  if (params.encryption) {
+    if (params.useApi) {
+      // Currently only CLI support for testing encryption
+      callback(new Error('UNSUPPORTED: cannot abort API backups at this time.'));
```

Comment:
Wrong err msg here? `UNSUPPORTED: cannot perform encrypted API backups at this time.`